text/en/chapters/algorithms.md

```diff
@@ -108,6 +108,10 @@ But here's another program that implements exactly the same algorithm, this time
 {image filename="highscore-in-scratch.png" alt="High score program in Scratch"}
 
 Both of the above programs are the same algorithm.
+Programming languages are often created or adapted to express algorithms clearly for a specific problem domain.
```

Comment:
From my experience, K2 words are fine, and realistically you need them. But non jargon K3 can generally be replaced with simpler words.